### ISSUE TITLE
About the initial weights

### ISSUE THREAD

**Issue by lh641446825** [kanezaki/rotationnet#8]
Sorry for bothering you. I download initial weights for fine-tuning the models, but when I unzip the package, I got the following information:

> gzip: stdin: not in gzip format
> tar: Child returned status 1
> tar: Error is not recoverable: exiting now

I found that the website `http://www.cs.berkeley.edu/~rbg/r-cnn-release1-data.tgz`  did not have the `r-cnn-release1-data.tgz` package. Can you provide the updated website?

**Comment by kanezaki** [comment]
I uploaded the necessary file here:
https://www.dropbox.com/s/9h7xel1nl5gc0lc/ilsvrc_2012_train_iter_310k?dl=0

Thanks!

**Comment by lh641446825** [comment]
Thanks! I downloaded the weights. When I input the command` /home/caffe-rotationnet2/build/tools/caffe train -solver Training/rotationnet_modelnet40_case2_solver.prototxt -weights caffe_nets/ilsvrc_2012_train_iter_310k 2>&1 | tee log.txt`, I got the following information:

> F0418 08:04:43.886162  6901 layer_factory.hpp:81] Check failed: registry.count(type) == 1 (0 vs. 1) Unknown layer type: ImageData (known types: AbsVal, Accuracy, ArgMax, BNLL, BatchNorm, BatchReindex, Bias, Concat, ContrastiveLoss, Convolution, Crop, Data, Deconvolution, Dropout, DummyData, ELU, Eltwise, Embed, EuclideanLoss, Exp, Filter, Flatten, HDF5Data, HDF5Output, HingeLoss, Im2col, InfogainLoss, InnerProduct, Input, LRN, LSTM, LSTMUnit, Log, MVN, MemoryData, MultinomialLogisticLoss, MyAccuracy, MySoftmax, MySoftmaxLoss, MySoftmaxWithLoss, PReLU, Parameter, Pooling, Power, Python, RNN, ReLU, Reduction, Reshape, SPP, Scale, Sigmoid, SigmoidCrossEntropyLoss, Silence, Slice, Softmax, SoftmaxWithLoss, Split, TanH, Threshold, Tile)
> *** Check failure stack trace: ***
>     @     0x7f675a802daa  (unknown)
>     @     0x7f675a802ce4  (unknown)
>     @     0x7f675a8026e6  (unknown)
>     @     0x7f675a805687  (unknown)
>     @     0x7f675ae33774  caffe::LayerRegistry<>::CreateLayer()
>     @     0x7f675ae38d0c  caffe::Net<>::Init()
>     @     0x7f675ae3ae05  caffe::Net<>::Net()
>     @     0x7f675af6ceaa  caffe::Solver<>::InitTrainNet()
>     @     0x7f675af6dfac  caffe::Solver<>::Init()
>     @     0x7f675af6e2da  caffe::Solver<>::Solver()
>     @     0x7f675af52f63  caffe::Creator_SGDSolver<>()
>     @           0x40f48e  caffe::SolverRegistry<>::CreateSolver()
>     @           0x4084e4  train()
>     @           0x405e4c  main
>     @     0x7f6759573f45  (unknown)
>     @           0x406753  (unknown)
>     @              (nil)  (unknown)

What I should do?

**Comment by kanezaki** [comment]
I'm not sure about that error... Have you compiled [caffe-rotationnet2](https://github.com/kanezaki/caffe-rotationnet2) with no error?

**Comment by lh641446825** [comment]
Yes, I compiled caffe-rotationnet2 with no error.  I got the following information:

> CXX/LD -o .build_release/examples/cpp_classification/classification.bin
> CXX examples/cifar10/convert_cifar_data.cpp
> CXX/LD -o .build_release/examples/cifar10/convert_cifar_data.bin
> CXX examples/siamese/convert_mnist_siamese_data.cpp
> CXX/LD -o .build_release/examples/siamese/convert_mnist_siamese_data.bin
> CXX/LD -o python/caffe/_caffe.so python/caffe/_caffe.cpp
> touch python/caffe/proto/__init__.py
> PROTOC (python) src/caffe/proto/caffe.proto

**Comment by kanezaki** [comment]
Understood. These lines in _Makefile.config_ should be left commented out.
```
# uncomment to disable IO dependencies and corresponding data layers
# USE_OPENCV := 0
# USE_LEVELDB := 0
# USE_LMDB := 0
```

**Comment by lh641446825** [comment]
Sorry. I comment out  `# USE_OPENCV := 0`, then compile caffe-rotationnet2. I got  this error :
```
LD -o .build_release/lib/libcaffe.so.1.0.0-rc3
/usr/bin/ld: cannot find -lopencv_imgcodecs
collect2: error: ld returned 1 exit status
make: *** [.build_release/lib/libcaffe.so.1.0.0-rc3] Error 1
```
I can't find how to solve it. In this [issue](https://github.com/BVLC/caffe/issues/4490) someone said uncomment `USE_OPENCV := 0`

**Comment by kanezaki** [comment]
Please install OpenCV 2 or 3.
https://github.com/BVLC/caffe/wiki/Ubuntu-16.04-or-15.10-Installation-Guide

**Comment by lh641446825** [comment]
Sorry, I have installed OpenCV before.
